src/Controllers/WeatherForecastController.cs

```diff
@@ -29,7 +29,7 @@ public WeatherForecastController(ISecretProvider secretProvider, ILogger<Weather
         [HttpGet]
         public async Task<IEnumerable<WeatherForecast>> Get()
         {
-            string secret = await _secretProvider.GetRawSecretAsync("MySecret");
+            string secret = await _secretProvider.GetRawSecretAsync("Logging:LogLevel:Default");
```

Comment:
Let's stick with good examples ;)

```suggestion
            string secret = await _secretProvider.GetRawSecretAsync("Weather-API-Key");
```